holidays/groups/international.py

```
@@ -212,7 +212,7 @@ def _add_womens_day(self, name):
         """
         return self._add_holiday_mar_8(name)
 
-    def _add_world_war_two_victory_day(self, name, is_western=True):
+    def _add_world_war_two_victory_day(self, name, *, is_western=True):
```

Ruff (0.14.0)

Missing return type annotation for private function `_add_world_war_two_victory_day`

(ANN202)